flow/scripts/floorplan.tcl

```diff
@@ -87,6 +87,34 @@ if {[info exists ::env(MACRO_WRAPPERS)]} {
   }
 }
 
+##### Restructure for timing #########
+if { [info exist ::env(RESYNTH_TIMING_RECOVER)] && $::env(RESYNTH_TIMING_RECOVER) == 1 } {
+  repair_timing
```

Comment:
Should remove_buffers be above this block?

Reply:
Wanted to re-use the buffers inserted by yosys and do timing opt on top of that. remove_buffers before this would increase run time also. Since these results are only to get estimate of timing, this way would be a bit faster.